crates/ggml/README.md

```diff
@@ -0,0 +1,139 @@
+# GGML - Large Language Models for Everyone
+
+[GGML](https://github.com/ggerganov/ggml) is a format for distributing large
+language models (LLMs); the name is a portmanteau of the initials of its
```

Comment:
Suggest mentioning that GGML is a library for ML stuff first, and then mention the format. The format can also be used for any ML thing that targets GGML-the-library (including whisper.cpp), not strictly LLMs, but that's where it's taken off.